cmd/podman/system_renumber.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	renumberCommand     cliconfig.SystemRenumberValues
+	renumberDescription = `
+        podman system renumber
+
+        Migrate lock numbers to handle a change in maximum number of locks
```

Comment:
I miss some explanation of why and when this is necessary. Is it something users must to do or an optional migration? The more details the better :)